### PR TITLE
add moyo header

### PR DESCRIPTION
**Pull request by AnoriFrell**
[DEMO LINK] (https://AnoriFrell.github.io/layout_moyo-header/)
[TEST REPORT LINK] (https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,32 @@
...
+        <nav class="navigation">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
+            <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+            <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+            <li class="nav__item">
+              <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover">
+              Laptops & computers
```
Comment (volodymyr-soltys97) [inline src/index.html:28]: Consider using title case for the link text 'Laptops & computers' to match other link texts

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,32 @@
...
+          <a href="#Home">
+            <img src="images/logo.png" alt="Moyo icon">
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
+            <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+            <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+            <li class="nav__item">
```
Comment (volodymyr-soltys97) [inline src/index.html:24]: Replace '&amp;' with '&amp;' in the href attribute for proper encoding

File: src/index.html [hunk 3]
```diff
@@ -9,6 +9,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="nav__wraper">
```
Comment (volodymyr-soltys97) [inline src/index.html:13]: Change the class name 'nav__wraper' to 'nav__wrapper' for better readability

File: readme.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://AnoriFrell.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:4]: Add these links to PR description

File: src/index.html (hunk 1 above)
Comment (volodymyr-soltys97) [inline src/index.html:28]: Remember about code style
```suggestion
              <a
                href="#laptops&computers"
                class="nav__link"
                data-qa="hover"
              >
                Laptops & computers
```

File: src/index.html [hunk 5]
```diff
@@ -9,6 +9,32 @@
...
+        <div class="nav__icon">
+          <a href="#Home">
+            <img src="images/logo.png" alt="Moyo icon">
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
```
Comment (volodymyr-soltys97) [inline src/index.html:21]: Fix this with code style
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>
```
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more perfect

File: src/index.html (hunk 2 above)
Comment (AnoriFrell) [reply src/index.html:24]: Guess, it means to replace '#' with '&', right? But why?

**Comment by AnoriFrell** [comment]
[DEMO LINK](https://AnoriFrell.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)

**Review by AnoriFrell — COMMENTED** [review]
[DEMO LINK](https://AnoriFrell.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)

**Review by AnoriFrell — COMMENTED** [review]
[DEMO LINK](https://AnoriFrell.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)

**Review by AnoriFrell — COMMENTED** [review]
[DEMO LINK](https://AnoriFrell.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)

**Review by AnoriFrell — COMMENTED** [review]
[DEMO LINK](https://AnoriFrell.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://AnoriFrell.github.io/layout_moyo-header/report/html_report/)

File: readme.md (hunk 4 above)
Comment (volodymyr-soltys97) [reply readme.md:4]: Still not added, you should to add these links to the PR description

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,69 @@
...
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:57]: Create a variable for this color and use it everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Let's improve your code

**Pull request closed without merging**